docs/scenarios/k8s_lib_injection.md

```diff
@@ -24,7 +24,7 @@ The lib-injection project is a feature to allow injection of the Datadog library
 into a customer's application container without requiring them to modify their
 application images.
 
-This feature enables applications written in Java, Node, Python, DotNet or Ruby running
+This feature enables applications written in Java, Node, Python, .NET or Ruby running
```

Comment:
Node or Node.js ?

Reply:
Ah, I didn't consider searching just for `Node`. Honestly, I'm ok with `Node` as a shorthand for `Node.js`. As long as people don't write `NodeJs` or something like that, I'm happy. I've searched now and changed `Node` to `Node.js` in the places where it made sense and will push a fix ASAP